src/offlinelogstorage/dlt_offline_logstorage_behavior.c

```diff
@@ -664,13 +710,37 @@ DLT_STATIC int dlt_logstorage_find_last_dlt_header(void *ptr,
     return -1;
 }
 
+/**
+ * dlt_logstorage_write_to_log
+ *
+ * Write logdata to log storage file
+ *
+ * @param ptr       A pointer to the data to write
+ * @param size      The size of the data blocks
+ * @param nmemb     The number of blocks to write
+ * @param config    A pointer to DltLogStorageFilterConfig
+ */
+DLT_STATIC int dlt_logstorage_write_to_log(void *ptr, size_t size, size_t nmemb,
+                                           DltLogStorageFilterConfig *config)
+{
+#ifdef DLT_LOGSTORAGE_USE_GZIP
+    if (config->gzip_compression) {
+        return gzfwrite(ptr, size, nmemb, config->gzlog);
```

Comment:
As far as I know, gzfwrite is added since 2017.
Hence, using it here may cause incompatible to some old machines (Ex: Ubuntu 16). Could use please consider using gzwrite as an alternative?

Reply:
That is an alternative. But is this really an issue? What's the use case for building dlt-daemon on a 7 year old out of service and support linux distribution?

Reply:
Using this API is fine with our use cases. So we can keep it.
Ubuntu 16.04 is still supported on paid plans, but if people really want to use it they can deactivate this new feature.